docker/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM alpine:3.6
+
+MAINTAINER Ian Denhardt (@zenhack)
+
+ARG SIMP_LE_VERSION=master
+
+ENV PATH=/usr/local/bin:$PATH \
+    SIMP_LE_VERSION=${SIMP_LE_VERSION}
+
+RUN apk --no-cache add bash openssl python3 && \
+    apk --no-cache --virtual .build-deps add gcc git libffi-dev musl-dev openssl-dev py-pip python3-dev && \
+    pip3 install --no-cache-dir --upgrade pip setuptools wheel && \
+    pip3 install --no-cache-dir git+https://github.com/zenhack/simp_le.git@$SIMP_LE_VERSION && \
+    apk --update --purge del .build-deps
+
+COPY ./entrypoint.sh /simp_le/entrypoint.sh
+
+WORKDIR /simp_le/certs
+
+ENTRYPOINT [ "/bin/bash", "/simp_le/entrypoint.sh" ]
+CMD [ "--help" ]
```

Comment:
I'm not super comfortable with this much copypasta; is there a way to factor some of this out?

Reply:
Can you be more specific ? Is it the presence of two Dockerfiles or something inside this one ?

Reply:
The fact that there are two (very similar) Dockerfiles. Sorry if that wasn't clear.

Reply:
I have an idea how to combine the two, I'll test it and come back to you.

If it doesn't work we can stil have only one Dockerfile provided we use Dockercloud's automated build instead of Dockerhub's because of a missing feature in the latter.

The two alternatives have their own small downsides I'll detail later.